src/shared/ui/image/utils/checkForSuitable.ts

```diff
@@ -0,0 +1,19 @@
+const checkForSuitable = (imgUrl: string, width: number = 430) => {
```

Comment:
Maybe it will be better to create constant for `430`?

Reply:
The file has changed looks different. I would leave empty lines to separate the logic in in new one)